app/logic/Mail/ActiveSync/ActiveSyncAccount.ts

```diff
@@ -95,19 +97,28 @@ export class ActiveSyncAccount extends MailAccount {
 
     await this.listFolders();
 
+    // `listFolders` will subscribe to new user address books,
+    // but until #155 is fixed we need to link existing ones manually.
     for (let addressbook of appGlobal.addressbooks) {
       if (addressbook.protocol == "addressbook-activesync" && addressbook.url.startsWith(this.url + "?") && addressbook.username == this.username) {
         (addressbook as ActiveSyncAddressbook).account = this;
         await (addressbook as ActiveSyncAddressbook).listContacts();
       }
     }
 
+    // `listFolders` will subscribe to new user calendars,
+    // but until #155 is fixed we need to link existing ones manually.
     for (let calendar of appGlobal.calendars) {
       if (calendar.protocol == "calendar-activesync" && calendar.url.startsWith(this.url + "?") && calendar.username == this.username) {
         (calendar as ActiveSyncCalendar).account = this;
         await (calendar as ActiveSyncCalendar).listEvents();
       }
     }
+
+    // ActiveSync doesn't have streaming notifications, instead it
+    // can tell us when a pingable needs to resync again. There's no
```

Comment:
"it can tell us"

Can you please briefly (in a few words, with helpful pointers) describe *how* it tells us?